.github/actions/gar-auth/action.yaml

```diff
@@ -13,9 +14,8 @@ runs:
       name: "Authenticate to Google Cloud"
       uses: "google-github-actions/auth@v0"
       with:
-        create_credentials_file: false
         token_format: "access_token"
-        workload_identity_provider: ${{ inputs.GCP_WORKLOAD_IDENTITY_PROVIDER }}
+        workload_identity_provider: projects/865717136349/locations/global/workloadIdentityPools/github-actions/providers/github-actions
```

Comment:
I couldn't figure out a way to keep this provider thing "secret" but I think that's fine since it's not really a secret anyways.